apps/codeforafrica/src/payload/collections/Projects.js

```diff
@@ -0,0 +1,122 @@
+import dateTime from "../fields/dateTime";
+import image from "../fields/image";
+import linkArray from "../fields/links/linkArray";
+import linkGroup from "../fields/links/linkGroup";
+import richText from "../fields/richText";
+import slug from "../fields/slug";
+import tags from "../fields/tags";
+import nestCollectionUnderPage from "../utils/nestCollectionUnderPage";
+
+const Projects = {
+  slug: "projects",
+  admin: {
+    useAsTitle: "name",
+  },
+  fields: [
+    {
+      name: "name",
+      label: {
+        en: "Name",
+      },
+      type: "text",
+      required: true,
+    },
+    {
+      name: "tagLine",
+      label: {
+        en: "Tag Line",
+      },
+      type: "text",
+    },
+    {
+      name: "title",
```

Comment:
What's the difference between `name` and `title`?

Reply:
@Kilemensi, from the implementation [here](https://github.com/CodeForAfrica/ui/blob/develop/apps/codeforafrica/content/projects/academyafrica.md) both title and name exist and they are both different.

Reply:
Yes @koechkevin hence me asking what's their difference?

Reply:
This is how it is the difference from the site @kilemensi 
![image](https://github.com/CodeForAfrica/ui/assets/39160236/96398e0f-8cd0-4439-8bad-e2d38e2245f5)

Reply:
Very weird seeing name and title used this way @koechkevin ... feels like title should be name, subtitle should be title, etc.

... but, lets continue while we think about it some more.